.eslintrc.json

```diff
@@ -6,7 +6,7 @@
     "es2018": true
   },
   "parserOptions": {
-    "ecmaVersion": 2018,
+    "ecmaVersion": 2020,
```

Comment:
We want to stick to ECMA 2018 so please don't change this line. Add an exception via a comment to the lines where eslint complaints.

Reply:
Unfortunately it's an eslint parsing error rather than a rule violation so the usual way to add an exception for the line doesn't work. The only alternative I know of would be to ignore these files as a whole in the eslint config, like this:
```json
  "ignorePatterns": [
    "examples/jsm/loaders/DRACOLoader.js",
    "examples/jsm/loaders/KTX2Loader.js"
  ],
```
But that does mean we don't get the benefit of linting for the entirety of those files. Let me know if you'd like me to proceed on that basis.

Reply:
Then let's exclude both files from linting. 

BTW: What ECMA 2020 syntax are you using? We have to be aware that this syntax does not break bundlers like WebPack 4. That would be a red flag that makes the PR impossible to merge.